### PR TITLE
display no apps/functions message

### LINKED ISSUE #257: The "deploy one" button is redundant
When there are no applications deployed, in the Applications view, there is a button that says "deploy one". This button is redundant, as we already have the one on the right labeled as "Deploy New App".

We should remove the "deploy one" one.

### PR DESCRIPTION
**Pull request by sameersbn**
Apps:

no-apps

has-apps

Functions:

no-functions

has-functions

closes #257

### CODE REVIEW

**Comment by arapulido** [comment]
@sameersbn I don't think we should add an Error if there are no applications or functions. It is not really an error, it is just empty :) @Angelmmiguel any ideas?

**Comment by arapulido** [comment]
@sameersbn Looks good to me! The only thing is that I would use "Deploy New App" in both buttons to avoid the inconsistency with Functions

Also, would like @prydonius review in any case

File: dashboard/src/components/FunctionList/FunctionList.tsx [hunk 1]
```diff
@@ -52,19 +52,47 @@ class FunctionList extends React.Component<IFunctionListProps> {
...
+            )}
           </div>
           <hr />
         </header>
-        {this.props.error ? this.renderError() : <CardGrid>{chartItems}</CardGrid>}
+        {this.props.error ? (
+          this.renderError()
+        ) : this.props.functions.length === 0 ? (
+          <MessageAlert
+            header={"Unleash the power of Kubeless"}
```
Comment (prydonius) [inline dashboard/src/components/FunctionList/FunctionList.tsx]: catchy ;)

**Comment by prydonius** [comment]
> The only thing is that I would use "Deploy New App" in both buttons to avoid the inconsistency with Functions

Oh just noticed this, I agree with @arapulido except I would say we can drop the "New" in "Deploy New Function", and ideally also add a + icon or similar. We can improve that in another PR.

**Comment by Angelmmiguel** [comment]
Hello! Sorry, I missed read the first comment. However, I see you used the [Alerts](https://design.bitnami.com/category/Components/Feedback/#Alerts) component :+1:

**Pull request merged**